plugins/imagespacingbox/plugin.js

```diff
@@ -14,23 +14,65 @@
 						return;
 					}
 
+					var getFocusedWidget = function (editor) {
+						var widget = editor.widgets.focused;
+
+						if (widget && widget.name == 'image') return widget;
```

Comment:
I'm not sure this should be a function: I know it's been called in two places, but since we're returning `null` we could probably inline this and just check for `editor.widgets.focused`

If we really want to keep this as a function though, we should

1. Use strict equality comparison here (`===` and not `==`) 
2. Add some curly braces on that `if` statement

I know these aren't required to make it work, but since we apply those rules in all our frontend related code, I think it makes sense to apply them here too